<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigsObjNewSkillDialog</class>
 <widget class="QDialog" name="ConfigsObjNewSkillDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Sobre a habilidade</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Habilidade:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QComboBox" name="comboBoxSkill"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Objeto:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QComboBox" name="comboBoxSkillObject">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Intensidade:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QSpinBox" name="spinBoxIntensity">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Custo de magia:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2" colspan="2">
       <widget class="QSpinBox" name="spinBoxMagicCost">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="5">
       <widget class="QCheckBox" name="checkBoxMagicAttack">
        <property name="text">
         <string>É ataque mágico (usa inteligência)</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Som:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEditSound">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QToolButton" name="toolButtonSound">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QToolButton" name="toolButtonRemoveSound">
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxObtain">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Forma de obter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxAlways">
        <property name="text">
         <string>Sempre</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBoxLevel">
        <property name="text">
         <string>Nivel:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBoxLevel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBoxGetItem">
        <property name="text">
         <string>Coletar item:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxProviderObject">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
